src/retry.decorator.ts

```diff
@@ -76,11 +70,14 @@ export function Retryable(options: RetryOptions): DecoratorFunction {
 
 export class MaxAttemptsError extends Error {
   code = '429';
-  /* if target is ES5, need the 'new.target.prototype'
-  constructor(msg?: string) {
-      super(msg)
-      Object.setPrototypeOf(this, new.target.prototype)
-    } */
+  public retryCount: number;
+  public originalError: Error; 
+  constructor(originalError: Error, retryCount: number) {
```

Comment:
since this is a `public/exported` class, I guess it would be better to make them optional to keep backward comparability. What do you think?